Makefile

```diff
@@ -18,9 +18,7 @@ BINNAME := yay
 PACKAGE := ${PKGNAME}_${VERSION}_${ARCH}
 
 ifneq (,$(findstring gccgo,$(GOCC)))
```

Comment:
Is this ifneq still needed? I don't remember it being needed before moving to modules.

Reply:
It builds but it will always complains of `go build: when using gccgo toolchain, please pass compiler flags using -gccgoflags, not -gcflags` and I think it ignores the arguments after because this: `-X main.version=${VERSION}'` is not supported by gcc-go.

I would like to remove that ifneq but I haven't found a suitable solution